### ISSUE TITLE
More contrast between text and background

### ISSUE THREAD

**Issue by otuxe** [okestonia/jkan#396]
There has to be more contrast between text and background. Helps to improve web accessibility.

**Comment by andreskytt** [comment]
Based on https://contrastchecker.com/ the current colours of #777 and #FFF give a ratio of 20.14 which exceeds the WCAG guideline of 18. What should the new colours be?

**Comment by andreskytt** [comment]
Upon closer inspection, what is probably the issue is the grey-background box on the front page which indeed does not meet the criteria for fonts below 18pts and the colour difference criteria. It does not concern the actual content, however and we are supposed to get a brand new design for the portal anyway?
